misk-service/src/main/kotlin/misk/ServiceGraphBuilder.kt

```diff
@@ -6,13 +6,16 @@ import com.google.common.util.concurrent.ServiceManager
 import com.google.inject.Key
 import misk.CoordinatedService.Companion.CycleValidity
 import com.google.inject.Provider
+import wisp.logging.getLogger
 
 /**
  * Builds a graph of [CoordinatedService]s which defer start up and shut down until their dependent
  * services are ready.
  */
 internal class ServiceGraphBuilder {
   private var serviceMap = mutableMapOf<Key<*>, CoordinatedService>()
+
+  // A map of downstream services -> their upstreams.
```

Comment:
```suggestion
  /** A map of downstream services -> their upstreams. */
```

Probably can use proper Kotlin docs since this is a val not within a function.